### PR TITLE
Bump ember to `1.11.0`

### PR DESCRIPTION
**Pull request by seanpdoyle**
:tada:

### CODE REVIEW

**Comment by fivetanley** [comment]
@seanpdoyle can you rebase?

**Comment by seanpdoyle** [comment]
@fivetanley done

**Comment by fivetanley** [comment]
let's give 11.1 a shot maybe? seems weird for container stuff.

**Comment by wecc** [comment]
Do we still need the `handlebars` dependency?

File: bower.json [hunk 1]
```diff
@@ -2,16 +2,12 @@
   "name": "ember-data",
   "private": true,
   "dependencies": {
-    "ember": "~1.10.0"
+    "ember": "~1.11.1"
   },
   "devDependencies": {
     "qunit": "~1.17.0",
     "jquery": "~1.10.x",
-    "handlebars": ">= 2.0.0 < 3.0.0",
```
Comment (seanpdoyle) [inline bower.json]: :v: 

@wecc

**Comment by igorT** [comment]
Travis fails?

**Comment by igorT** [comment]
Rebased as #2993

**Pull request closed without merging**